app/views/categories/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<p style="color: green"><%= notice %></p>
+
+<div class='container-amount'>
+  <h1 class='title-categories'>Categories</h1>
+  <p class='total-amount'> Total Amount $<%= Category.total_amount(@categories) %></p>
+</div>
+<h3 class='new-category'><%= link_to "New category", new_category_path %></h3>
+<div class="categories">
+  <% @categories.each do |category| %>
+  <div class='category-render'>
+    <%= render category %>
+    <p class='details-but'><%= link_to "Details category", category %></p>
+  </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Good job working on this page, but it does not follow the original design guidelines. Please kindly ensure it follows the original design guidelines.
- [ ] Also, all categories are supposed to be added to the categories index page.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/221278717-3a5d3e5a-7e3f-4076-be64-edfc8ee12b79.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/221278643-83633be5-bb66-4d5b-a621-ae39631041f6.png)